### ISSUE TITLE
Prettier CSS (especially for wall-of-text-pages) would be nice to have

### ISSUE THREAD

**Issue by wilhelmy** [irchelp/wio#85]
Unfortunately I can't do CSS, so YOU (Yes, YOU!) go do it.

**Comment by nih16agr** [comment]
I tried prettifying all css files, please have a look

**Comment by wilhelmy** [comment]
Thanks, just to clarify, this is just formatting of the *.css files, you didn't make any visual changes, correct?

I was referring to the look of the page but I'm considering merging this as well. Thanks!

**Comment by nih16agr** [comment]
Yes, I have only formatted the code, I have not at all changed the visuals and look of pages.
I even tested it locally. Thank you :)
